frontend/src/components/ShootStatus.vue

```diff
@@ -40,15 +40,15 @@ limitations under the License.
         <div>{{ tooltipText }}</div>
       </v-tooltip>
     </div>
-    <template slot="content-after">
-      <pre v-if="!!popperMessage" class="alert-message">{{ popperMessage }}</pre>
+    <template slot="content">
+      <pre v-if="!!popperMessage" class="alert-message" v-html="popperMessage"></pre>
```

Comment:
may be an extra component for ansiHTML display?

Reply:
done